core/platform-common/src/main/java/org/sunbird/mail/SendgridConnection.java

```diff
@@ -53,6 +55,11 @@ public Transport createConnection(RequestContext context) {
       props.put("mail.smtp.auth", "true");
       props.put("mail.smtp.port", port);
 
+      if(Is_TLS_Enable =="true")
```

Comment:
property may be set to "TRUE" 
change the if condition as follows.
if("true".equalsIgnoreCase(isTlsEnabled))